neighbors/src/components/containers/formsContainer.jsx

```diff
@@ -0,0 +1,12 @@
+import { Login } from '../login/Login';
+import { Registration } from '../RegistrationForm/RegistrationForm';
+import '../mainLayout.css';
+
+export function FormsContainer() {
+  return (
+    <div className="signin-signup">
+      <Login></Login>
+      <Registration></Registration>
```

Comment:
Jeżeli komponent nie ma `children` to powinnen być self-closing, np:
`<Login></Login>` -> `<Login />`